plane/source/docs/quicktune.rst

```diff
@@ -0,0 +1,60 @@
+
+The advantage over QAUTOTUNE is that QUICKTUNE is safer because the vehicle does not need to move or twitch and QAUTOTUNE can occasionally result in instablity. Therefore QAUTOTUNE is not longer recommended for use, except by expects.
+
+The disadvantage is that QUICKTUNE cannot find the vehicle's maximum rotational accelerations or tune the attitude angle outer loops from their default or pre-tune adjusted (:ref:`quadplane-vtol-tuning-process`) values. However, tuning only the inner rate loops usually will yield a stable, safe flying vehicle.
+
+Setup
+=====
+- Set :ref:`QWIK_ENABLE<QWIK_ENABLE>` = "1" to enable QUICKTUNE.
+- Set :ref:`QWIK_AXES<QWIK_AXES>` to which axes (Roll/Pitch/Yaw).
+
+Activation RC Switch
+--------------------
+Set up a two or three position RC switch with `RCx_OPTION`` = "181". If it is a two position switch set :ref:`QWIK_OPTIONS<QWIK_OPTIONS>` bit 0 to "1", and you probably want to set the :ref:`QWIK_AUTO_SAVE<QWIK_AUTO_SAVE>` parameter to autosave the tune once completed.
+
+With a three position switch, moving the switch to "high" (>1800us) is the "save" position which will save the current tune parameters, even if the tune is not completed yet. Moving the switch to the middle position activates the tuning process ("tune"). Moving the switch to the "low" position (<1200us), stops the tuning and reverts the parameters to their values before tuning began.
+
+.. note:: RC AUX switches only operate on transitions of the switch. Their position at boot does nothing, so if a switch is "high" at boot, in order to get the "high" function, it must be moved to another position and then back "high".
+
+With a two position switch, "high" is the "tune" position, and "low" stops/reverts tuning. In order to save parameters, you must have set the :ref:`QWIK_AUTO_SAVE<QWIK_AUTO_SAVE>` parameter or use a GCS AUX switch facility.
+
+Tuning Flight
+=============
+- Wait for a calm day and go to an open area with good GPS reception
+- Use an OSD, TX OSD (using :ref:`Yaapu <common-frsky-yaapu>`), or GCS. This is where output from the tune will appear
+- Move the RC switch to the low position OR push MP's Aux Function's "Low" button
+- Arm and takeoff in Loiter mode and climb to a height of about 3m
+- Begin the tune by moving the RC switch to the "tune" position OR push MP's Aux Function's "Mid" button
+- Monitor the progress of the tune using the GCS's Messages tab
+- If necessary reposition the vehicle using the RC transmitter.  This will temporarily pause tuning and restore the original gains.  Tuning will resume a few seconds after the RC sticks are returned to their center position
+- If the vehicle begins oscillating violently cancel the tune by moving the RC switch to the low position OR push MP's Aux Function's "Low" button
```

Comment:
the tune will automatically abort if an angle error over QWIK_ANGLE_MAX is detected